### ISSUE TITLE
Add new package "obspy"

### ISSUE THREAD

**Issue by krischer** [ContinuumIO/anaconda-issues#369]
Hi all,

would it be possible to add our ObsPy package to the official Anaconda distribution? It is the biggest Python package for seismology with a (for seismology) fairly large user base:

http://obspy.org
https://github.com/obspy/obspy
https://pypi.python.org/pypi/obspy

The conda receipe can be found here: https://github.com/obspy/obspy/tree/master/misc/installer/anaconda

We also build ObsPy package for all platforms: https://anaconda.org/obspy/obspy

Furthermore all of its dependencies are already part of the official Anaconda distribution so adding it should be very simple.

Thanks a lot!

Lion

**Comment by groutr** [comment]
I would be happy to look at adding obspy into the official anaconda repositories.

**Comment by krischer** [comment]
Great :-) Let me know if I can assist in any way!

**Comment by groutr** [comment]
Is this entry point in console scripts supposed to be empty?
``` 'obspy-mseed-recordanalyzer = '```

**Comment by cossatot** [comment]
Is it python 3 compatible yet?  I remember earlier this year having to make a new 2.7 environment so that I could use it.  It'd be great if this wasn't necessary any more.

**Comment by krischer** [comment]
Yes - the current version is compatible with Python 2.6, 2.7, 3.3, and 3.4. The next one will drop support for Python 2.6.

**Comment by groutr** [comment]
I'm building the recipe and I'm getting several failures/errors from the test suite.  Is that expected?

**Comment by krischer** [comment]
Not really. Can you paste the errors here?

**Comment by krischer** [comment]
Any update on this or any way we could help to make this happen?
